project1.html

```diff
@@ -2,40 +2,46 @@
 <head>
 	<link rel="stylesheet" type="text/css" href="project1.css">
 	<link rel="stylesheet" type="text/css" href="normalize.css">
+	
 </head>
 	<header>
 		<nav class="navbar">
 			<input class="nav-button" type="button" value="About">
 			<input class="nav-button" type="button" value="Portfolio">
+			<img src="liam.png" alt="myname">
 			<input class="nav-button" type="button" value="Contact">
+			<input class="nav-button" type="button" value="Download CV">
 		
```

Comment:
it's better to use instead of for these - since they are not part of a form, the tag isn't the best to use. it's also useful to use because that way you can make them scroll down to the section you want, whereas if you wanted to do that with a button you would need javascript